apps/member-profile/app/routes/_profile.directory.tsx

```diff
@@ -253,47 +257,83 @@ function FilterDirectoryDropdown() {
 
   return (
     <Dropdown.Container onClose={onClose}>
-      <IconButton
+      {(filter === 'general') ? <IconButton
         backgroundColor="gray-100"
         backgroundColorOnHover="gray-200"
         icon={<Filter />}
         onClick={onClick}
         shape="square"
-      />
+      /> : <Button onClick={onClick}>
+          {toTitleCase(filter)}
+        </Button>}
+
+      {/* I tried to make the buttons for companies and schools look the same as the icon button
+      but I couldn't get the color to match */}
```

Comment:
Looks good as it is! Let's remove this comment. Also, in general let's leave these types of comments in the PR rather than in the code